src/targetingPackages/DLL/microsoft.targetingpack.netframework.v4.7.2/1.0.0/microsoft.targetingpack.netframework.v4.7.2.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>$(ArtifactsBinDir)Microsoft.Targetingpack.Netframework.v4.7.2/1.0.0/</OutputPath>
+    <BaseIntermediateOutputPath>$(ArtifactsObjDir)Microsoft.Targetingpack.Netframework.v4.7.2/1.0.0/</BaseIntermediateOutputPath>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <NuspecFile>microsoft.targetingpack.netframework.v4.7.2.nuspec</NuspecFile>
```

Comment:
Are both of the nuspecs added here required?  They look identical.

Reply:
Right. I have the DLL directory to mimic the structure to be copied to artifacts during the build. As part of #65 effort. This will be taken care of. Ill try to combine src/ and DLL/ packages to one.